static/css/components/toast.less

```diff
@@ -25,6 +28,7 @@
 
   .toast--close {
     padding-left: .75em;
+    pointer-events: auto;
```

Comment:
Set this on the `.toast`; we still want folks to be able to eg select the toast message.
```suggestion
```